.kokoro/system_tests.sh

```diff
@@ -64,5 +64,11 @@ fi
 # Install global test dependencies
 composer install -d testing/
 
+# cd into specific subdirectory (if appropriate)
+MAIN_DIR=$(pwd)
+if [[ -z "${SUB_DIRECTORY}" ]]; then
```

Comment:
@ace-n kokoro tests are failing on [this line](https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/testing/run_test_suite.sh#L79), which runs `git diff`.  It's because this bash test is incorrect, and as a result the script is running `cd` and ending up in the home directory.

Try this instead:

```suggestion
if [ ! -z "${SUB_DIRECTORY}" ]; then
```